local-env/setup-kind.sh

```
@@ -0,0 +1,84 @@
+#!/bin/bash
+# vim:sts=2:ts=2:et:sw=2:tw=0
+
+readonly script_dir=$(dirname "$0")
+readonly kind_cmd="${1}"
+readonly kind_cluster="${2}"
+readonly kind_node_version="${3}"
+readonly kind_kubeconfig="${4}"
```

We should either fail fast or provide a default if these are not provided.